centrallix/expression/exp_functions.c

```diff
@@ -3671,6 +3946,9 @@ exp_internal_DefineFunctions()
 	xhAdd(&EXP.Functions, "log10", (char*)exp_fn_log10);
 	xhAdd(&EXP.Functions, "power", (char*)exp_fn_power);
 	xhAdd(&EXP.Functions, "pbkdf2", (char*)exp_fn_pbkdf2);
+	xhAdd(&EXP.Functions, "levenshtein", (char*)exp_fn_levenshtein);
+	xhAdd(&EXP.Functions, "fuzzy_compare", (char*)exp_fn_fuzzy_compare);
+	xhAdd(&EXP.Functions, "similarity", (char*)exp_fn_similarity);
```

Comment:
I'm considering renaming fuzzy_compare() and similarity() to more descriptive names, but we have another branch that depends on this right now, so I'm not making that change yet.